### DISCUSSION TITLE
hugo server总是报错

### DISCUSSION THREAD

**Discussion by lubover** [hugo-next/hugo-theme-next#102] [Q&A]
win10系统，用的extended版的，按照   #13 步骤来的

> $ hugo new site hugo-next-exmaple
$ cd hugo-next-exmaple
$ git init
$ git submodule add https://github.com/hugo-next/hugo-theme-next.git themes/hugo-theme-next
$ cp -r themes/hugo-theme-next/exampleSite/* .
$ rm -rf config.toml
$ hugo server

报错如下：

> $ hugo server
Start building sites …
hugo v0.114.1-e9b716ad9869b79e7c374bbdae4daf5ebd4+extended windows/amd64 Bu
ildDate=2023-06-23T11:02:58Z VendorInfo=gohugoio

> Built in 55 ms
Error: error building site: assemble: "F:\Hugo\sites\hugo-next-exmaple\content\p
ost\custom-files.md:36:1": failed to extract shortcode: template for shortcode "
note" not found

**Comment by elkan1788** [comment]
@lubover 上面的错误看起来是引用了`NEXT`当前不支持的`shortcode`， 建议可以使用 [hugo-theme-next-starter](https://github.com/hugo-next/hugo-theme-next-starter) 快速体验和构建自己的站点。

**Reply by lubover** [reply]
OK,成了
